dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -757,7 +757,7 @@ CREATE TABLE `t_ds_resources` (
   `create_time` datetime DEFAULT NULL COMMENT 'create time',
   `update_time` datetime DEFAULT NULL COMMENT 'update time',
   `pid` int(11) DEFAULT NULL,
-  `full_name` varchar(64) DEFAULT NULL,
+  `full_name` varchar(128) DEFAULT NULL,
```

Comment:
Please also add it to postgres and upgrade sql.

Reply:
Has been updated

Reply:
You need to create a folder ```2.0.6_schema`` like ```dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.4_schema```. And put ```ALTER TABLE t_ds_resources MODIFY COLUMN NAME VARCHAR(128);``` in ```mysq/dolphinscheduler_ddl.sql```. Same as ```postgresql```.

Reply:
It has been modified, please review it again